xml/installation-installation-configure_3par.xml

```diff
@@ -158,14 +152,19 @@
     <para>
      Edit the
      <literal>~/openstack/my_cloud/config/nova/kvm-hypervisor.conf.j2</literal>
-     file add this line under the <literal>[libvirt]</literal> section:
+     file and add this line under the <literal>[libvirt]</literal> section:
     </para>
     <para>
      Example:
     </para>
 <screen>[libvirt]
 ...
 iscsi_use_multipath=true</screen>
+    <para>
+     Additionally, if you are planning on attaching an encrypted 3PAR volume
```

Comment:
Probably can remove "additionally" here too. Just "If you want to attach an encrypted 3PAR..."

Reply:
Done